src/current/_includes/releases/v23.1/v23.1.17.md

```diff
@@ -0,0 +1,103 @@
+## v23.1.17
+
+Release Date: March 19, 2024
+
+{% include releases/release-downloads-docker-image.md release=include.release %}
+
+<h3 id="v23-1-17-security-updates">Security updates</h3>
+
+- A user with the `VIEWACTIVITY` privilege can now request a statement bundle from the [**Statements** page](https://www.cockroachlabs.com/docs/v23.1/ui-statements-page) in the DB Console or with the internal builtin function `crdb_internal.requests_statement_bundle`. [#119638][#119638]
+- Clusters using [Cluster Single Sign-on (SSO) with JSON web tokens (JWTs)](https://www.cockroachlabs.com/docs/v23.1/sso-sql) can now optionally fetch signing keys from configured issuers instead of configuring static signing keys for each issuer. When the new cluster setting `server.jwt_authentication.jwks_auto_fetch.enabled` is set to `true`, signing keys are automatically fetched from the issuer using metadata published in its OpenID configuration. In this case, static signing keys in `server.jwt_authentication.jwks` are ignored. When automatic fetching is enabled, there may be a slight increase in network latency for each JWT authentication request, proportional to the latency between the cluster and the issuer's endpoint. [#120063][#120063]
+- DB Console cookies are marked `Secure` for the browser when the cluster is running in secure mode. [#119262][#119262]
+- The DB Console `session` cookie is now marked `HttpOnly` to prevent it from being read by any Javascript code. [#119262][#119262]
+
+<h3 id="v23-1-17-sql-language-changes">SQL language changes</h3>
+
+- The new [cluster setting](https://cockroachlabs.com/docs/v23.1/cluster-settings) `server.max_open_transactions_per_gateway`, when set to a non-negative value, prevents users without the `admin` role from executing a query if more than this number of transactions are open on the current gateway node. [#118949][#118949]
+- Added support for [index hinting](https://www.cockroachlabs.com/docs/v23.1/table-expressions#force-index-selection) with `INSERT` and `UPSERT` statements. This allows `INSERT ... ON CONFLICT` and `UPSERT` queries to use index hints in the same way as `UPDATE` and `DELETE` statements. [#119600][#119600]
```

Comment:
mentions of INSERT, UPSERT, UPDATE, DELETE could all be links to those statements' doc pages